node/src/events.rs

```diff
@@ -272,6 +312,29 @@ impl fmt::Display for NodeEvent {
             NodeEvent::FatalDaserError { error } => {
                 write!(f, "Daser stopped because of a fatal error: {error}")
             }
+            NodeEvent::NewSubjectiveHead { height } => {
+                write!(f, "New subjective head: {height}")
+            }
+            NodeEvent::AddedHeaderFromHeaderSub { height } => {
+                write!(f, "Added header {height} from HeaderSub")
```

Comment:
```suggestion
                write!(f, "Added header {height} from header-sub")
```
I think protocol name uses this format, but up to u